src/main/kotlin/uk/gov/justice/digital/hmpps/approvedpremisesapi/service/PlacementApplicationService.kt

```diff
@@ -341,6 +349,36 @@ class PlacementApplicationService(
 
     placementDateRepository.saveAll(placementDates)
     baselinePlacementApplication.placementDates = placementDates
```

Comment:
is it worth adding some logging here so we can verify that this legacy behaviour is discontinued when we expect? 

e.g. 
`WARNING: Adding multiple dates to placement application for application {applicationId}. This behaviour is deprecated.`